src/lib/api-client.ts

```diff
@@ -42,7 +42,9 @@ class APIClient {
       headers: {
         "Content-Type": "application/json",
         Accept: "application/json",
-        "X-UA": `@topsort/sdk ${version}`,
+        "X-UA": config.userAgent
+          ? `${config.userAgent}`
```

Comment:
Could we somehow concatenate both values. So that in case a client overrides the user agent we still know which version of the sdk they are using.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Also note that the header should be something like topsort-sdk/version.

In case we get a user agent in the config it could be 

topsort-sdk/version custom-user-agent

Reply:
This is how Marcio had it by default but I find it somewhat intrusive. We can suggest to users to do it but I don't think we should force it on them

Reply:
Slightly different than what I thought, I've update to this way, but with the @ before
`@topsort-sdk/version custom-user-agent`

What you guys think?

Reply:
Since this is for us I think its good to keep the first version/what you suggest.